contractsV5/IFeeHandler.sol

```diff
@@ -1,5 +1,5 @@
-pragma solidity 0.5.11;
-
-interface IFeeHandler {
-    function handleFees(address[] calldata eligibleWallets, uint[] calldata rebatePercentages) external payable returns(bool);
-}
+pragma solidity 0.5.11;
+
+interface IFeeHandler {
+    function handleFees(address[] calldata eligibleWallets, uint[] calldata rebatePercentages) external payable returns(bool);
+}
```

Comment:
Seems like an invisible change here... don't know what's the difference.

Reply:
totally invisible.

try to copy past and make it go away.
if not will merge.

Reply:
Copy pasted, git doesn't show that the file changed.